.github/workflows/qa.yaml

```diff
@@ -5,26 +5,26 @@ on:
   pull_request:
     branches: [ "main" ]
 jobs:
-  make-versions:
+  python-versions:
```

Comment:
This just spruces up the code a little bit. We know it's working because the QA build is passing against all these versions of Python.